lib/pages/category_list_item.dart

```diff
@@ -70,9 +77,7 @@ class _CategoryListItemState extends State<CategoryListItem>
       end: BorderRadius.zero,
     ).animate(_controller);
 
-    _isExpanded = PageStorage.of(context)?.readState(context) as bool ??
```

Comment:
@rami-a @JoseAlba Any idea why PageStorage is used for this particular widget? It currently does not support state restoration, but it seems like it's unnecessarily used here.

Reply:
I don't actually recall. I think it's probably safe to remove it